static/app/components/modals/editOwnershipRulesModal.tsx

```diff
@@ -14,7 +14,9 @@ type Props = ModalRenderProps & EditOwnershipRulesModalOptions;
 const EditOwnershipRulesModal = ({Body, Header, onSave, ...props}: Props) => {
   return (
     <Fragment>
-      <Header closeButton>{t('Edit Ownership Rules')}</Header>
+      <Header closeButton>
+        <h4>{t('Edit Ownership Rules')}</h4>
```

Comment:
Should we change the styling on whatever the modal Header does

Reply:
yeah we definitely should but not in this PR, old bean

Reply:
Do you want to actually change this to a h4 then, how many other form `Header`'s have h4s in them?

Reply:
Not in this PR

Reply:
I mean are you sure you want it change this one here, when we might update all of them at once later?

I'm just arguing for consistency